### DISCUSSION TITLE
老丈人的三轮车🛺 | 连波的闲谈杂鱼

### DISCUSSION THREAD

**Discussion by giscus** [godruoyi/gblog#7] [Announcements]
# 老丈人的三轮车🛺 | 连波的闲谈杂鱼

老丈人有一辆三轮车🛺，每次不开车回去时，我都能坐上老丈人的三轮车。以前，我和阿宝会在镇上下大巴，老丈人就从家开着三轮车来接我们。

https://godruoyi.com/posts/dads-tricycle/

**Comment by TheSaltwaterRoom** [comment]
多写点

**Reply by godruoyi** [reply]
谢谢，在写了，😂

**Comment by zippo-zu** [comment]
精神世界极大丰富
